### ISSUE TITLE
Staging Review finding: Heading level when submitted forms can't be displayed

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#94354]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Veteran Facing Forms
**Product:** Forms Status
**Feature:** Form Status on My VA & Confirmation Page

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that does not meet best practices.](https://design.va.gov/about/experience-standards/#user-encounters-content-that-meets-best-practices)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On My VA, when form statuses and drafts can't be displayed, there's a warning alert that's displayed in the "Benefit applications and forms" section with heading "We can't access your benefit applications and forms right now." Both the heading for this section of the page and the heading for the alert are H2s.

This could be confusing to screen reader users navigating by heading, since the warning alert is content within the "Benefit applications and forms" section rather than its own new section.

<details><summary>Link, screenshot or steps to recreate</summary>

!["Benefit applications and forms" heading followed by a warning alert with heading "We can't access your benefit applications and forms right now."](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/80272-Brian-3.png?raw=true)

</details>

## Recommended action

The warning alert should have an H3 heading rather than an H2, which is consistent with the heading structure for content when it is able to be displayed.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
